apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -1194,6 +1195,13 @@ def _ingest_uns_dict(
         _maybe_set(parent, parent_key, coll, use_relative_uri=use_relative_uri)
         coll.metadata["soma_tiledbsoma_type"] = "uns"
         for key, value in dct.items():
+
+            # Things like uns/_scvi cannot be uploaded to TileDB Cloud since
+            # "name must begin with an unaccented alphanumeric character" --
+            # so turn that into "u_scvi".
+            if not re.match('^[a-zA-Z].*$', key):
```

Comment:
```python
import string
# ...
# for bonus points maybe pull this concatenation to the module level
if key[0] not in string.ascii_letters + string.digits:
  key = "u" + key
```

Also: Should this only be done for group metadata entries? Like, would it make sense if `_neighbors` was instead just `"bob"` that it should be `_neighbors: bob` in the metadata?

Reply:
@thetorpedodog re the former: done

Re the latter: I don't know but I have no signal from "data in the wild" that we should do so.